app/views/providers/means/vehicle_details/_form.html.erb

```diff
@@ -30,9 +30,8 @@
                                             legend: { text: t(".remaining_payments.question"), tag: "h2", size: "m" },
                                             hint: { text: t(".remaining_payments.detail_of_payments_to_include") }) do %>
         <%= form.govuk_radio_button(:payments_remain, true,
-                                    link_errors: true,
```

Comment:
Isn't this still required in order for the error summary to link to the first radio button?

Reply:
I removed it because I think it's a holdover from when this was_on it's own page. Now it's the second of 4 or 5 questions and will only ever have explicit responses. I think we normally only need this on list of optional questions on a page.
I'll confirm no degradation on the branch though 👍